### PR TITLE
feat: For OrderProposal updates, check the order-proposals feed

### PR DESCRIPTION
**Pull request by lukehesluke**

### CODE REVIEW

File: packages/openactive-broker-microservice/app.js [hunk 1]
```diff
@@ -1147,85 +1147,6 @@ app.post('/assert-unmatched-criteria', function (req, res) {
   }
 });
 
-// /**
```
Comment (lukehesluke) [inline packages/openactive-broker-microservice/app.js:1150]: i commented out this stuff in a recent PR when i realised it was obsolete

File: packages/openactive-integration-tests/test/helpers/flow-helper.js [hunk 2]
```diff
@@ -89,60 +92,22 @@ class FlowHelper {
 
   P = pMemoize(async () => {
     throw new Error('This should not be called');
-    // this._assertStageShouldNotBeSkipped('P');
```
Comment (lukehesluke) [inline packages/openactive-integration-tests/test/helpers/flow-helper.js:92]: Same again. By removing the obsolete RequestHelper.getOrder function, I had to remove its callees as well (and so on)

File: packages/openactive-broker-microservice/app.js (hunk 1 above)
Comment (nickevansuk) [reply packages/openactive-broker-microservice/app.js:1150]: Nice shall we just delete it? It was left in on the assumption the test suite was still using it somewhere

File: packages/openactive-broker-microservice/validator/validator-worker.js [hunk 3]
```diff
@@ -1,10 +1,13 @@
 // workers/auth.js
 const { validate } = require('@openactive/data-model-validator');
 const { expose } = require('threads/worker');
+const { silentlyAllowInsecureConnections } = require('../src/util/suppress-unauthorized-warning');
 
 // Note this is duplicated between app.js and validator.js, for efficiency
 const VALIDATOR_TMP_DIR = './tmp';
 
+silentlyAllowInsecureConnections();
```
Comment (nickevansuk) [inline packages/openactive-broker-microservice/validator/validator-worker.js:9]: Added this to stop the random warnings that come out of the harvester presently

File: packages/openactive-integration-tests/test/helpers/flow-helper.js (hunk 2 above)
Comment (nickevansuk) [reply packages/openactive-integration-tests/test/helpers/flow-helper.js:92]: Great to see, shall we just bin this code also?

Is there a reason why we couldn't go further and delete `BAfterP` etc. if they have no references in the code base?

File: packages/openactive-integration-tests/test/helpers/flow-stages/flow-stage-recipes.js [hunk 4]
```diff
@@ -255,7 +255,8 @@ const FlowStageRecipes = {
       orderFeedUpdateParams: {
         ...defaultFlowStageParams,
         prerequisite: p,
-        testName: 'Order Feed Update (after Simulate Seller Approval)',
+        testName: 'OrderProposal Feed Update (after Simulate Seller Approval)',
+        orderType: 'order-proposals',
```
Comment (nickevansuk) [inline packages/openactive-integration-tests/test/helpers/flow-stages/flow-stage-recipes.js:259]: Should this be renamed `orderFeedType`? (As orderType would imply `Order` or `OrderProposal` standard OA term values)

**Review by nickevansuk — APPROVED** [review]
Great to see us culling the old code - a few minor suggestions

File: packages/openactive-broker-microservice/app.js (hunk 1 above)
Comment (lukehesluke) [reply packages/openactive-broker-microservice/app.js:1150]: Sorry I meant that I commented it out in a recent PR and that this PR deletes it

File: packages/openactive-integration-tests/test/helpers/flow-helper.js (hunk 2 above)
Comment (lukehesluke) [reply packages/openactive-integration-tests/test/helpers/flow-helper.js:92]: This is deleted. I intend to go further for flow-helper & request-state once some other stuff is done

File: packages/openactive-integration-tests/test/helpers/flow-stages/flow-stage-recipes.js (hunk 4 above)
Comment (lukehesluke) [reply packages/openactive-integration-tests/test/helpers/flow-stages/flow-stage-recipes.js:259]: Good idea 👍

Comment (lukehesluke) [reply packages/openactive-integration-tests/test/helpers/flow-stages/flow-stage-recipes.js:259]: This is done

**Review by nickevansuk — APPROVED** [review]

File: packages/openactive-broker-microservice/app.js (hunk 1 above)
Comment (nickevansuk) [reply packages/openactive-broker-microservice/app.js:1150]: Ha! So you did - I didn't see the red text background for some reason...

**Pull request merged**